src/App.js

```diff
@@ -18,6 +20,9 @@ export default function App() {
           <Route path="/volunteer-opportunities">
             <VolunteerOpportunities />
           </Route>
+          <Route path="/pages/graduates-summer-2019">
```

Comment:
React pages should not live under the `/pages` path, that is for static HTML pages.

Reply:
I think that pathname is messing up the React page. When I look at the PR preview, it seems that these two files are identical:

https://deploy-preview-225--refcode.netlify.app/pages/graduates-summer-2019
https://deploy-preview-225--refcode.netlify.app/pages/graduates-summer-2019.html

Reply:
yes. they are identical. but the first one is a react component. it is the optimized one.

Reply:
oh. I see the preview shows the same file. it is not showing the react one.

Reply:
I think it is counting that the first one is also the 2019.html. I had to remove the /pages part.

Reply:
I just removed /pages part. now we can see how optimized the images are when we inspect on the images.

Reply:
https://deploy-preview-225--refcode.netlify.app/pages/graduates-summer-2019

Reply:
Ah yes! Now I see. For example:

https://deploy-preview-225--refcode.netlify.app/static/media/IMG_20190720_181823-320.jpg
https://deploy-preview-225--refcode.netlify.app/static/media/IMG_20190720_181823-640.jpg
https://deploy-preview-225--refcode.netlify.app/static/media/IMG_20190720_181823-960.jpg
https://deploy-preview-225--refcode.netlify.app/static/media/IMG_20190720_181823-1200.jpg
https://deploy-preview-225--refcode.netlify.app/static/media/IMG_20190720_181823-1800.jpg
https://deploy-preview-225--refcode.netlify.app/static/media/IMG_20190720_181823-2400.jpg

This is really cool!!!!!!!!!!!!